libcloud/common/exceptions.py

```diff
@@ -68,7 +72,11 @@ def exception_from_message(code, message, headers=None):
         'headers': headers
     }
 
-    if headers and 'retry_after' in headers:
-        kwargs['retry_after'] = headers['retry_after']
+    if headers and 'retry-after' in headers:
+        http_date = parsedate_tz(headers['retry-after'])
```

Comment:
Have you confirmed this returns `None` on parse error instead of throwing an exception?

Reply:
Yes, as described on the docs, parsedate_tz() returns None or a tuple.